google/cloud/aiplatform/metadata/constants.py

```diff
@@ -26,3 +26,8 @@
     SYSTEM_EXPERIMENT: _DEFAULT_SCHEMA_VERSION,
     SYSTEM_METRICS: _DEFAULT_SCHEMA_VERSION,
 }
+
+# GCP UI currently only displays experiment context which has a "experiment_deleted" key and False value.
+# This is a temporary solution to integrate tensorboard results into GCP UI.
+UI_DETECTION_KEY = "experiment_deleted"
```

Comment:
nit: EXPERIMENT_DELETED_KEY maybe? Let's do not couple this to UI

Reply:
@jialuzh, we will also need to set another field: experiment_type = MB. Can you add it as part of this PR?

Reply:
done